packages/ui/src/__tests__/atoms/Avatar.test.tsx

```diff
@@ -1,13 +1,27 @@
-import { render, screen } from '@testing-library/react'
+import { fireEvent, render, screen } from '@testing-library/react'
+import { vi } from 'vitest'
 import { Avatar } from '../../components/atoms/Avatar'
 
+const mockClick = vi.fn()
+const defaultAvatarCss = {
+  width: '97px',
+  height: '97px',
+  borderRadius: '50%',
+}
 describe('Avatar', () => {
   it('should render with given props', () => {
-    render(<Avatar src="/test.png" alt="users portrait" />)
+    render(<Avatar src="/test.png" alt="users portrait" avatarCss={defaultAvatarCss} onClick={mockClick} />)
 
     const image: HTMLImageElement = screen.getByRole('img')
+
     expect(image).toBeInTheDocument()
     expect(image).toHaveAttribute('src', '/test.png')
     expect(image).toHaveAttribute('alt', 'users portrait')
+   
+    expect(image).toHaveStyle('width: 97px; height: 97px; border-radius: 50%;');
+
+    fireEvent.click(image)
+    expect(mockClick).toHaveBeenCalled()
+
   })
```

Comment:
This test is ok, but also include one first with the component by default, without optional props, please.

Reply:
Test without passing css props, done, thansk